brainglobe_utils/image_io/load.py

```diff
@@ -112,7 +112,7 @@ def load_any(
             sort=sort_input_file,
             n_free_cpus=n_free_cpus,
         )
-    elif src_path.endswith((".tif", ".tiff")):
+    elif src_path.suffix == ".tif" or src_path.suffix == ".tiff":
```

Comment:
```suggestion
    elif src_path.suffix in [".tif", ".tiff"]:
```

I am a mathematician so an naturally going to suggest the set-like notation and slightly fewer keystrokes :sweat_smile: